src/site/_static/css/custom.css

```diff
@@ -1,14 +1,13 @@
 /*
     Title and background colors for the class activity.
 */
-.rst-content .activity .admonition-title {
+/*  .rst-content .activity .admonition-title {
```

Comment:
These seem unrelated.

Reply:
i must keep branching from not main. Will become moot once other PR merges.